Assets/Plugins/iOS/Shopify/BuyTests/ApplePayFlowTests.swift

```diff
@@ -74,9 +73,9 @@ class ApplePayFlowTests: XCTestCase {
         
         let expectation      = self.expectation(description: "MockAuthorizationController.invokeDidSelectShippingContact failed to complete")
         let expectedSubtotal = PKPaymentSummaryItem(label: "SUBTOTAL", amount: 25.47)
-        let expectedShipping = PKPaymentSummaryItem(label: "SHIPPING", amount: 8.44)
+        let expectedShipping = PKPaymentSummaryItem(label: "SHIPPING", amount: standardShippingMethod.amount)
         let expectedTaxes    = PKPaymentSummaryItem(label: "TAXES",    amount: 2.93)
-        let expectedTotal    = PKPaymentSummaryItem(label: "TOTAL", amount: 33.91)
+        let expectedTotal    = PKPaymentSummaryItem(label: "TOTAL", amount: 31.30)
```

Comment:
it's `standardShippingMethod.amount` here, because we expect it to select the first shipping method. We used `expeditedShippingMethod.amount` above because we selected `expeditedShippingMethod`